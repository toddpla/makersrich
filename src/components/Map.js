import React, { Component } from 'react';
import map from '../assets/maps/proofofwork.png'

class Map extends Component {

  render() {
    return (
      <div className="map"
        style={{backgroundImage: `url('${map}')`,
                height: '672px',
                maxHeight: '672px',
                width: '1024px',
                maxWidth: '1024px',
                overflow: 'auto',
                position: 'relative',
                display: 'block',
                margin: 'auto'
              }}>

      </div>
    );
  }

}

export default Map;
