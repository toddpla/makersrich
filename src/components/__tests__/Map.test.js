import React from 'react'
import renderer from 'react-test-renderer';
import { shallow } from 'enzyme'
import Map from '../Map'

test('renders correctly', () => {
  const wrapper = renderer.create(<Map />).toJSON();
  expect(wrapper).toMatchSnapshot();
});
