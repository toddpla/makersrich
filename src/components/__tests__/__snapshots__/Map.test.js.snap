// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`renders correctly 1`] = `
<div
  className="map"
  style={
    Object {
      "backgroundImage": "url('proofofwork.png')",
      "display": "block",
      "height": "672px",
      "margin": "auto",
      "maxHeight": "672px",
      "maxWidth": "1024px",
      "overflow": "auto",
      "position": "relative",
      "width": "1024px",
    }
  }
/>
`;
